### PR TITLE
feat: add upgrade-16 (proposal 75) 

### LINKED ISSUE #167: Add upgrade-16 (proposal 75) to agoric-3-proposals
## What is the Problem Being Solved?

[proposal 75](https://agoric.explorers.guru/proposal/75) to `agoric-upgrade-16` passed.

agoric-3-proposals should be updated to reflect this.

## Description of the Design

As a final step in release 16 (this should get added to a runbook template), the proposal from a3p-integration should be moved to agoric-3-proposals.

Move upgrade-16 a3p-integration proposal to a3p repo, and use final docker tags and upgrade info

### PR DESCRIPTION
**Pull request by dckc**
fixes #167 

Initial attempts to test locally failed at the `yarn build` step:

```
ERROR: docker exporter does not currently support exporting manifest lists
...
Instead use a builder that supports multiplatform such as depot.dev.
```

cc @turadg @Chris-Hibbert @LuqiPan

### CODE REVIEW

**Comment by turadg** [comment]
>ERROR: docker exporter does not currently support exporting manifest lists

Remove the `platforms` property from package.json to build for just the local platform.

File: proposals/75:upgrade-16/provisionPool.test.js
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/provisionPool.test.js:1]: I think provisioning was part of U16, but provisionPool was not. (Yeah they're distinct things.)

File: proposals/75:upgrade-16/upgradeVaults.js
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/upgradeVaults.js:1]: Vaults was not included in U16

File: proposals/75:upgrade-16/use.sh
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/use.sh:1]: not in upgrade 16

File: proposals/75:upgrade-16/synthetic-chain-excerpt.js
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/synthetic-chain-excerpt.js:1]: Please double-check this. My impression was that `synthetic-chain` didn't make it into U16, but i'm not certain.

File: proposals/75:upgrade-16/exitOffer.js
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/exitOffer.js:1]: I do not know whether this test pertains to U16.

File: proposals/75:upgrade-16/provisionPool.test.js
Comment (turadg) [reply proposals/75:upgrade-16/provisionPool.test.js:1]: indeed https://github.com/Agoric/agoric-sdk/tree/dev-upgrade-16/a3p-integration/proposals/a%3Aupgrade-16

Comment (dckc) [reply proposals/75:upgrade-16/provisionPool.test.js:1]: Apparently I copied from the wrong source. The right source is:

https://github.com/Agoric/agoric-sdk/tree/agoric-upgrade-16/a3p-integration/proposals/a%3Aupgrade-16/

File: proposals/75:upgrade-16/vatDetails.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import dbOpenAmbient from 'better-sqlite3';
```
Comment (dckc) [inline proposals/75:upgrade-16/vatDetails.js:1]: I wonder why this file is here. I can't find anything that imports from it.

Comment (Chris-Hibbert) [reply proposals/75:upgrade-16/vatDetails.js:1]: It's used in the vaults and auctions upgrade. I dunno whether any parts were used in orchestration upgrades.

**Comment by dckc** [comment]
I gather ci for this is a bit fragile. I'll try it again.

Symptoms:

```
  ✘ Timed out while running tests
```
https://github.com/Agoric/agoric-3-proposals/actions/runs/10372747065/job/28716195667?pr=166#step:11:12725

**Comment by dckc** [comment]
@turadg @Chris-Hibbert it's green. Please take a look.

File: proposals/75:upgrade-16/use.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (turadg) [inline proposals/75:upgrade-16/use.sh:1]: empty. consider dropping to aid debugging in the future (clear there's no "use" stage)

**Review by turadg — APPROVED** [review]

File: proposals/75:upgrade-16/probe-submission/probeZcfBundle-plan.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "probeZcfBundle",
+  "script": "probeZcfBundle.js",
+  "permit": "probeZcfBundle-permit.json",
+  "bundles": [
+    {
+      "entrypoint": "@agoric/vats/src/vat-zoe.js",
+      "bundleID": "b1-db507028460b0c9ad9b16a1e1094b4cd09cfbdc1779c992b42d828745788d98393ca4740599ac4b2e3a6f4bd8d9018aad6d509107eced9253fe7d3c0085709a7",
+      "fileName": "/home/connolly/.agoric/cache/b1-db507028460b0c9ad9b16a1e1094b4cd09cfbdc1779c992b42d828745788d98393ca4740599ac4b2e3a6f4bd8d9018aad6d509107eced9253fe7d3c0085709a7.json"
```
Comment (Chris-Hibbert) [inline proposals/75:upgrade-16/probe-submission/probeZcfBundle-plan.json:9]: do these local file paths on your machine matter?

Comment (dckc) [reply proposals/75:upgrade-16/probe-submission/probeZcfBundle-plan.json:9]: They don't affect execution.

I guess cleaning them up for looks is worthwhile...

**Pull request merged**